packages/test/test-end-to-end-tests/src/test/audience.spec.ts

```diff
@@ -212,4 +217,36 @@ describeCompat("Audience correctness", "FullCompat", (getTestObjectProvider, api
 			"client2's audience should be removed",
 		);
 	});
+
+	it("clientIdChanged & clientIdChanged event", async function () {
+		assert(apis.containerRuntime !== undefined);
+		if (apis.containerRuntime.version !== pkgVersion) {
+			// Only verify latest version of runtime - this functionality did not exist prior to RC3.
+			// Given that every version (from now on) tests this functionality, there is no reason to test old versions.
+			// This test does not use second container, so there is no need for cross-version tests.
+			this.skip();
+			return;
```

Comment:
This will still test the latest runtime version combined with older loader versions in the same container, and such kind of compat, right?

Reply:
Yes, we do want to test old loader & new runtime (to test back-compat polyfil in runtime). And yes, it will test it.